app/commands/github/task/sync_task.rb

```diff
@@ -0,0 +1,50 @@
+module Github
+  class Task
+    class SyncTask
+      include Mandate
+
+      initialize_with :issue
+
+      def call
+        if closed? || claimed?
+          destroy_task
+        else
+          create_or_update_task
+        end
```

Comment:
I prefer early return guards. Anything to avoid more indentation :)

```suggestion
        return destroy_task if closed? || claimed?
        
        create_or_update_task
```